src/gui/tracks/SampleTrackView.cpp

```diff
@@ -61,20 +61,22 @@ SampleTrackView::SampleTrackView( SampleTrack * _t, TrackContainerView* tcv ) :
 	m_mixerChannelNumber = new MixerChannelLcdSpinBox(2, getTrackSettingsWidget(), tr("Mixer channel"), this);
 	m_mixerChannelNumber->show();
 
-	m_volumeKnob = new Knob( KnobType::Small17, getTrackSettingsWidget(),
-						    tr( "Track volume" ) );
+	auto f = font();
+	f.setPixelSize(9);
```

Comment:
Can you use the adjustedToPixelSize instead of a direct call?

Reply:
Done with commit 39e1e9f862c.